targets/curie_bsp/setup.py

```diff
@@ -0,0 +1,195 @@
+    os.makedirs(location_path)
+  os.chdir(location_path)
+  # arc
+  if not os.path.islink(location_path + '/arc'):
+    os.symlink(jerry_path + '/targets/curie_bsp/jerry_app/arc', 'arc')
+
+  # include
+  if not os.path.islink(location_path + '/include'):
+    os.symlink(jerry_path + '/targets/curie_bsp/jerry_app/include', 'include')
+  # quark
+  if not os.path.islink(location_path + '/quark'):
+    os.symlink(jerry_path + '/targets/curie_bsp/jerry_app/quark', 'quark')
+
+  # jerryscript
+  location_path += '/quark'
+  os.chdir(location_path)
+  if not os.path.islink(location_path + '/jerryscript'):
+    os.symlink(jerry_path, 'jerryscript')
+
+
+# create Kbuild.mk
+def breadth_first_travel(root_dir):
+  out_put = ''
+  lists = os.listdir(root_dir)
+  dirs = []
+  for line in lists:
+    full_path = os.path.join(root_dir, line)
+    if os.path.isdir(full_path):
+      dirs.append(line)
+    else:
+      c_file = line.endswith('.c')
+      if c_file:
+        npos = line.find('.c')
+        out_put += 'obj-y += ' + line[0:npos] + '.o\n'
+        continue
+      asm_file = line.endswith('.S')
+      if asm_file:
+        npos = line.find('.S')
+        out_put += 'obj-y += ' + line[0:npos] + '.o\n'
+  for line in dirs:
+    out_put += 'obj-y += ' + line + '/\n'
+  os.chdir(root_dir)
+  file_to_be_created = open('Kbuild.mk', 'w+')
+  file_to_be_created.write(out_put)
+  file_to_be_created.close()
+  for line in dirs:
+    breadth_first_travel(os.path.join(root_dir, line))
+
+
+
+# create Kbuild.mk in jerryscript/
+def create_kbuild_in_jerryscript(jerry_path):
+  breadth_first_travel(jerry_path + '/jerry-core')
+  breadth_first_travel(jerry_path + '/jerry-libm')
+  #jerryscript/
+  out_put =  'obj-y += jerry-core/\n'
+  out_put += 'obj-y += jerry-libm/\n'
+  out_put += 'obj-y += targets/\n'
+  out_put += 'subdir-cflags-y += -DJERRY_NDEBUG\n'
+  out_put += 'subdir-cflags-y += -DJERRY_DISABLE_HEAVY_DEBUG\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_MEM_HEAP_AREA_SIZE=10*1024\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_NUMBER_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_STRING_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_BOOLEAN_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_ERROR_BUILTINS\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_ARRAY_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_MATH_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_JSON_BUILTIN\n'
+  #out_put += 'subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_ERROR_BUILTIN\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_LCACHE_DISABLE\n'
+  out_put += 'subdir-cflags-y += -DCONFIG_ECMA_PROPERTY_HASHMAP_DISABLE\n'
+  os.chdir(jerry_path)
```

Comment:
Do we really need to change dir? Can't we just join the path with the `Kbuild.mk` below and pass it that to the `open` ?